### ISSUE TITLE
How to create "custom Google Cloud project" (something changed?)

### ISSUE THREAD

**Issue by hershkoy** [iterative/dvc#4558]
I am trying to follow the instructions to generate a ["custom Google Cloud project"](https://dvc.org/doc/user-guide/setup-google-drive-remote#:~:text=Find%20and%20select%20the%20%22Google,Save%20(scroll%20to%20bottom).) ,  but it seems that something has changed in the google platform.  

The instructions says that  should choose "Other", but this option doesn't exist. See image below.

![image](https://user-images.githubusercontent.com/13259380/92807288-6a8cb500-f3c3-11ea-98fc-8bdfa34115a9.png)

What do I need to choose here?

**Comment by shcheklein** [comment]
Good point. Transferring this to the dvc.org repo. I think Desktop app is the right choice here. Would be great if you could edit the page- there is an Edit on Github button.
